Resources/public/javascript/cropper.js

```diff
@@ -0,0 +1,24 @@
+class Cropper {
+    constructor(item) {
+        this.item = item;
+        this.item.find($('*[data-method="getCroppedCanvas"]')).on('click', this.save.bind(this));
```

Comment:
Almost! 
find( **$(** '*[data-method="getCroppedCanvas"] '**)** )
should be without the **$()**
find('*[data-method="getCroppedCanvas"]')